.github/workflows/ci.yml

```diff
@@ -17,6 +17,11 @@ jobs:
       - run: npm install
       - run: npm run lint
       - run: npm run package:build
+      - uses: actions/setup-go@v3
```

Comment:
I think we ran into a transient error -- I need to look into it, but the merge into `main` passed, and this was the first failed check I've seen.

Reply:
> this was the first failed check I've seen

Ah, this is because the `clientcompat` test doesn't run in CI, but _only if_ the `CODECOV_TOKEN` environment variable is present:

https://github.com/tatethurston/TwirpScript/blob/413f01e3488fe9292fd7593d4e37ede21e0a4624/src/integration-tests/clientcompat/src/test.ts#L13-L16

My PRs are coming from a forked repo, so GitHub isn't providing their builds access to secrets (a security feature), so the `CODECOV_TOKEN` isn't getting set, hence the failure. 🙂

Reply:
🤦 I forgot about that hack. Sorry about that -- thank you for debugging this.